src/design-guidelines/action-buttons/action-buttons.md

```diff
@@ -0,0 +1,126 @@
+
+```tsx
+<limel-button
+    label="Ok"
+    slot="button"
+/>
+```
+:::
+
+<br/>
+
+*****
+
+## Primary versus secondary actions
+
+As you know, `limel-button` can have a `primary={true}` property, which makes it stand out by getting the primary color of your theme.
+
+One of the action buttons is normally the primary action. That is the action that you expect the users to take, or you want or prefer them to click on.
+
+<limel-example-action-buttons-primary-secondary></limel-example-action-buttons-primary-secondary>
+
+A primary button has a stronger visual weight, and appearing more distinct makes it easier for users to chose the correct, safe, or promoted action.
+
+You can use this visual distinction to promote the action that is safer, even if that action may not be the primary action. You can do so because sometimes:
+- it can be hard to identify the intention of the user
+- the action may be irreversible or risky
+
+<limel-example-action-buttons-primary-secondary-reversed></limel-example-action-buttons-primary-secondary-reversed>
+
+<br/>
+
+*****
+
+## Choosing good labels
+
+Labels you use should make sense together. You are usually dealing with contradictory actions. Therefore, what you put on each button should also reflect the contradiction.
+
+<limel-example-action-buttons-choosing-labels></limel-example-action-buttons-choosing-labels>
+
+Labels must be making each option as clear as possible. An explicit label give users more confidence in selecting the correct action.
+
+:::note
+- Make sure the label explains what the button does, rather than a generic label (such as “OK”, or "Yes").
+- Using verbs as labels instead of nouns is better, because a verb describes the action better.
+:::
+
+<limel-example-action-buttons-choosing-explicit-labels></limel-example-action-buttons-choosing-explicit-labels>
+
+<br/>
+
+*****
+
+## Providing a third alternative action
+
+Sometimes having two action is not enough for the user to take a decision with peace of mind. A common scenario is when a user has unsaved changes and tries to navigate away from the view. In such cases, typically a prompt maybe be shown to alert the user and give them a few logical choices.
+
+This prompt can offer two main actions of "Save" and "Discard" (which means "save & continue navigating away from this page" and "discard & continue navigating away from this page"), along with a third choice that allows users to "Cancel" (and go back / not continue navigating away!).
+
+<limel-example-action-buttons-third-alternative></limel-example-action-buttons-third-alternative>
+
+The action that takes users one step back inte the process is intentionally placed on the left side. This firstly makes it easier for users to mentally separate the other two, and secondly the placement makes it easier to connect the action to the notion of going back, in a UI which is designed for left-to-right script.
```

Comment:
```suggestion
The action that takes users one step back in the process is intentionally placed on the left side. This makes it easier for users to mentally separate the other two. Additionally, the placement makes it easier to connect the action to the notion of going back. (For left-to-right languages. Left and right will of course have reversed meaning in right-to-left languages, which is precisely why the CSS properties for these things use values like `flex-start` and `flex-end`, rather than `left` and `right`. See below.)
```

Reply:
scripts, not languages?

Reply:
```suggestion
The action that takes users one step back in the process is intentionally placed on the left side. This makes it easier for users to mentally separate the other two. Additionally, the placement makes it easier to connect the action to the notion of going back. (For left-to-right scripts. Left and right will of course have reversed meaning in right-to-left script, which is precisely why the CSS properties for these things use values like `flex-start` and `flex-end`, rather than `left` and `right`. See below.)
```